src/main.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App.jsx';
-import './index.css';
+import Layout from './components/Layout.jsx';
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
-    <App />
+    <Router>
+      <Layout>
+        <App />
+      </Layout>
+    </Router>
```

Comment:
we will not have this Layout all the time use Outlet component to differentiate  different layouts